CHANGELOG.md

```diff
@@ -1,5 +1,106 @@
 # Changelog
 
+## [v1.14.0](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/v1.14.0) (2023-12-19)
+
+[Full Changelog](https://github.com/mongodb/terraform-provider-mongodbatlas/compare/v1.13.1...v1.14.0)
+
+**Breaking changes:**
+
+- fix!: replaces .String\(\) method with internal method .TimeToString\(\) to align formating with Atlas API. [\#1699](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1699) ([marcosuma](https://github.com/marcosuma))
+   - Fixes date fields that were not compliant to the ISO 8601 timestamp format in UTC, in line with the documentation.
```

Comment:
not sure if this line has leading blanks

Reply:
Yes this is intentional as it is a nested bullet to provide more detail on the breaking change